docs/get-started/getting-started-databricks.md

```diff
@@ -50,9 +50,11 @@ The number of GPUs per node dictates the number of Spark executors that can run
 4. Databricks makes changes to the runtime without notification.
 
     Databricks makes changes to existing runtimes, applying patches, without notification.
-	[Issue-3098](https://github.com/NVIDIA/spark-rapids/issues/3098) is one example of this.  We run
-	regular integration tests on the Databricks environment to catch these issues and fix them once
-	detected.
+    [Issue-3098](https://github.com/NVIDIA/spark-rapids/issues/3098) is one example of this.  We run
+    regular integration tests on the Databricks environment to catch these issues and fix them once
+    detected.
+5. In Databricks 11.3, an incorrect result is returned for window frames defined by a range in case of DecimalTypes. 
```

Comment:
this sounds as if it's *always* the case. Let us add some caveat mentioning that this is an edge case, and only affects large decimals around the maximum precision probably causing range boundaries to be off

Reply:
Thanks for the review. I have made the change